README.md

```diff
@@ -3,129 +3,123 @@
-    <a href="https://github.com/schlagmichdoch/PairDrop/issues">Report Bug</a>
-    ·
-    <a href="https://github.com/schlagmichdoch/PairDrop/issues">Request Feature</a>
+    <a href="https://github.com/schlagmichdoch/PairDrop/issues">Request feature</a>
   </p>
 </div>
 
 ## Features
-[PairDrop](https://pairdrop.net) is a sublime alternative to AirDrop that works on all platforms.
-
-- File Sharing on your local network
-  - Send images, documents or text via peer to peer connection to devices on the same local network.
-- Internet Transfers
-  - Join temporary public rooms to transfer files easily over the internet!
-- Web-Application 
-  - As it is web based, it runs on all devices.
-
-You want to quickly send a file from your phone to your laptop?
-<br>You want to share photos in original quality with friends that use a mixture of Android and iOS?
-<br>You want to share private files peer to peer between Linux systems?
-<br>AirDrop is unreliable again?
-<br>_Send it with PairDrop!_
-
-Developed based on [Snapdrop](https://github.com/RobinLinus/snapdrop)
+File sharing on your local network that works on all platforms.
+
+- A multi-platform AirDrop-like solution that works.
+  - Send images, documents or text via peer-to-peer connection to devices on the same local network.
+- Internet transfers
+  - Join temporary public rooms to transfer files easily over the Internet.
+- Web-app 
+  - Works on all devices with a web browser (that supports HTML5, CSS 3, JS, and WebRTC).
+ 
+Send a file from your phone to your laptop?
+<br>Share photos in original quality with friends using Android and iOS?
+<br>Share private files peer-to-peer between Linux systems?
 
-## Differences to Snapdrop
-<details><summary>Click to expand</summary>
+## Differences to the [Snapdrop](https://github.com/RobinLinus/snapdrop) it is based on
+<details><summary>List view</summary>
 
-### Paired Devices and Public Rooms - Internet Transfer
-* Transfer files over the internet between paired devices or by entering temporary public rooms.
-* Connect to devices in complex network environments (public Wi-Fi, company network, Apple Private Relay, VPN etc.).
+### Paired Devices and Public Rooms — Internet Transfer
+* Transfer files over the Internet between paired devices or by entering temporary public rooms.
+* Connect to devices in complex network environments (public Wi-Fi, company network, Apple private relay, VPN, etc.).
 * Connect to devices on your mobile hotspot.
-* Devices outside your local network that are behind a NAT are connected automatically via the PairDrop TURN server.
+* Devices outside of your local network that are behind a NAT are autop-connected via the PairDrop TURN server.
 * Connect to devices on your mobile hotspot.
-* You will always discover devices on your local network. Paired devices and devices in the same public room are shown additionally.
+* Devices that are paired devices, in the same public room, or on your local network are shown.
```

Comment:
"Devices" is twice. Also, I think this changes the meaning of the sentence. Maybe we could do "Devices discover each other if they are either paired, in the same public room, or on the same local network."